build.gradle.kts

```diff
@@ -58,7 +58,9 @@ subprojects {
     implementation("org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE")
     implementation("org.postgresql:postgresql:42.3.5")
     implementation("org.liquibase:liquibase-core:4.10.0")
-
+    implementation("org.springframework.boot:spring-boot-starter-actuator:2.7.0")
```

Comment:
🤔 These dependencies will be applied to all sub-projects. If some of them are only used in the `platform` subproject, can we limit the scope?

Reply:
After looking into the code, seems like all you need in `core` is `io.micrometer:micrometer-registry-prometheus`.